packages/mask/src/plugins/Avatar/hooks/usePersonas.ts

```diff
@@ -0,0 +1,24 @@
+import { NextIDProof } from '@masknet/web3-providers'
+import { useAsyncRetry } from 'react-use'
+import { useCurrentVisitingIdentity, useLastRecognizedIdentity } from '../../../components/DataSource/useActivatedUI'
```

Comment:
Please don't access those hooks in this way. Learn this: https://github.com/DimensionDev/Maskbook/commit/5b94b2a18419d942346d8919d9f1913536dbf4a3

Reply:
This is not resolved. Please change.